aiohttp/client.py

```diff
@@ -130,7 +130,6 @@
     "request",
 )
 
-
```

Comment:
Please don't include formatting changes in the same PR that has functional changes.